### PR TITLE
Add release workflow

### PR DESCRIPTION
**Pull request by eddumelendez**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          jdkFile: ${{ steps.sdkman.outputs.file }}
+          server-id: bintray-awspring
+          server-username: BINTRAY_USERNAME
+          server-password: BINTRAY_PASSWORD
+      - name: Build with Maven
+        run: ./mvnw -V verify
+      - name: Build with Maven
+        run: ./mvnw -V deploy
+        env:
+          BINTRAY_USERNAME: awspring-releaser
```
Comment (eddumelendez) [inline .github/workflows/release.yml:41]: tbd /cc @maciejwalkowiak

Comment (maciejwalkowiak) [reply .github/workflows/release.yml:41]: The org name is awspring https://bintray.com/awspring

**Review by maciejwalkowiak — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (eddumelendez) [reply .github/workflows/release.yml:44]: this is the username to perform de deploy, can be you or me or a new user we want to create.

File: pom.xml [hunk 2]
```diff
@@ -291,4 +291,11 @@
 		</profile>
 	</profiles>
 
+	<distributionManagement>
+		<repository>
+			<id>bintray-awspring</id>
+			<url>https://api.bintray.com/maven/awspring/releases/spring-cloud-aws/;publish=1</url>
```
Comment (eddumelendez) [inline pom.xml:297]: format is `https://api.bintray.com/maven/awspring/{{repository}}/{{package}}/;publish=1` in terms of bintray.
/cc @maciejwalkowiak

Comment (maciejwalkowiak) [reply pom.xml:297]: 👍

File: .github/workflows/release.yml (hunk 1 above)
Comment (maciejwalkowiak) [reply .github/workflows/release.yml:44]: I guess it's the best to create a new user. Can you take care of it?

Comment (eddumelendez) [reply .github/workflows/release.yml:44]: sure!

**Pull request merged**